pkg/reconciler/pipelinerun/cancel_test.go

```diff
@@ -288,12 +286,12 @@ func TestCancelPipelineRun(t *testing.T) {
 						PipelineTaskName: "task-2",
 					},
 					{
-						TypeMeta:         runtime.TypeMeta{Kind: "Run"},
+						TypeMeta:         runtime.TypeMeta{Kind: pipeline.RunControllerName},
 						Name:             "r1",
 						PipelineTaskName: "run-1",
 					},
 					{
-						TypeMeta:         runtime.TypeMeta{Kind: "Run"},
+						TypeMeta:         runtime.TypeMeta{Kind: pipeline.CustomRunControllerName},
```

Comment:
```suggestion
						TypeMeta:         runtime.TypeMeta{Kind: pipeline.RunControllerName},
```

Reply:
also, can we please add `customRuns` to the `TestCancelPipelineRun` test?

Reply:
Done!